gnomad_qc/v4/create_release/create_de_novo_release.py

```diff
@@ -135,34 +135,50 @@ def get_releasable_de_novo_calls_ht(
     )
     mt = annotate_adj(mt)
 
-    # Approximate the AD and PL fields when missing.
+    # Many of our larger datasets have the PL and AD fields for homref genotypes
```

Comment:
```suggestion
    # Many of our larger datasets have the PL and AD fields for homref genotypes
```
can you check if this is true for v3 as well? if yes, it's best to say `v3 and v4 have the PL and ...`

Reply:
Yes, they are also NA in v3 VDS. FYI, I used this code to check: 
```
from gnomad_qc.v3.resources.basics import get_gnomad_v3_vds
from gnomad.sample_qc.relatedness import filter_to_trios

vds = get_gnomad_v3_vds(split=True, 
                       filter_intervals=['chr11:113409605-113475691'],
                       )

trios = hl.import_fam("gs://gnomad-qin/g1k.trios.fam", delimiter='\t')
vds = filter_to_trios(vds, trios)

mt = hl.vds.to_dense_mt(vds)
mt.entries().show(20)
```

Reply:
thank you for looking! this is good to know